api/envoy/admin/v3/listeners.proto

```diff
@@ -35,9 +35,8 @@ message ListenerStatus {
   // to listen on port 0, then this address has the port that was allocated by the OS.
   config.core.v3.Address local_address = 2;
 
-  // This is same with :ref:`local_address
-  // <envoy_v3_api_field_admin.v3.ListenerStatus.local_address>`. Just additional
-  // addresses that listener is listening on.
+  // The additional addresses the listener is listening on as specified via the :ref:`additional_addresses <...>`
```

Comment:
I didn't mean to actually use `...` here. Please fill in the actual ref link.

Reply:
hah, sorry about that, will fix